recipes/gnuradio-iio/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -ex
+
+mkdir build
+cd build
+
+# enable components explicitly so we get build error when unsatisfied
+cmake_config_args=(
+    -DCMAKE_BUILD_TYPE=Release
+    -DCMAKE_INSTALL_PREFIX=$PREFIX
+    -DLIB_SUFFIX=""
+    -DPYTHON_EXECUTABLE=$PYTHON
+    -DBoost_NO_BOOST_CMAKE=ON
+    -DGR_PYTHON_DIR=`$PYTHON -c "import site; print(site.getsitepackages()[0])"`
```

Comment:
There is a env variable for this, I think it is $SP_DIR: https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html

Reply:
How many times have I looked at that environment variables list and not noticed SP_DIR? Much cleaner, thanks!